.eslintrc

```diff
@@ -18,5 +19,8 @@
     "prefer-rest-params": 0,
     "prefer-reflect": 0,
     "valid-jsdoc": 0
+  },
+  "globals": {
+    "global": true
```

Comment:
IMO if the only place you need that global is in the one file, it's better to add it as a global there rather than in the main linting config. As it is now, you could easily use `global` anywhere without a linting warning when what you really want is to import your `global`-determining module.